src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+import jdk.jfr.Percentage;
+
+public class Task {
+    protected String description;
+    protected boolean isMarked;
+
+    public Task(String description) {
+        this.description = description;
+        this.isMarked = false;
+    }
+
+    public String getStatusIcon() {
+        return (isMarked ? "[/]" : "[ ]");
```

Comment:
Try using named constants to give context to these string literals (avoid usage of magic literals)